tests/phpunit/tests/post/attachments.php

```diff
@@ -519,4 +519,15 @@ public function test_wp_mime_type_icon_video() {
 
 		$this->assertStringContainsString( 'images/media/video.png', $icon );
 	}
+
+	/**
+	 * @ticket 60610
+	 */
+	public function test_wp_mime_type_icon_video_with_preferred_ext() {
+		$icon1 = wp_mime_type_icon( 'video/mp4', '.png' ); // Added `$preferred_ext` parameter.
+		$icon2 = wp_mime_type_icon( 'video/mp4', 'png' ); // Added `$preferred_ext` parameter without period.
+
+		$this->assertStringContainsString( 'images/media/video.png', $icon1 );
+		$this->assertStringContainsString( 'images/media/video.png', $icon2 );
+	}
```

Comment:
For both assertions, include the description argument so failed tests will include context. E.g.

`$this->assertStringContainsString( 'images/media/video.png', $icon1, 'Mime type icon correctly returned with ".png" argument.' );`

Reply:
@joedolson I've made the suggested changes.

Reply:
The second context string should be "returned with "png" argument'', but I'll fix that in the commit.